hardhat.config.ts

```diff
@@ -119,6 +119,9 @@ const config: HardhatUserConfig = {
     artifacts: "./out",
     sources: "./src/contracts",
   },
+  typechain: {
+    outDir: "./typechain-types",
```

Comment:
why is this necessary? isn't this the default?

Reply:
It is, weird.
 I got my types generated to `./types` a couple times